src/components/AppHeader/MobileMenu/MenuContent.tsx

```diff
@@ -1,4 +1,5 @@
 import { useState } from 'react';
+import { useDevice } from '@/hooks';
```

Comment:
here as well.. i think the one in hooks can be removed.. it was created before the usedevice in deriv-com/ui was implemented. also the test will break when using the one from deriv-com/ui.. need to mock it in the spec file